lib/elixir/lib/record.ex

```diff
@@ -222,16 +222,17 @@ defmodule Record do
       user(name: name) = record
       name #=> "meg"
 
-  By default, Elixir uses the record name as the first element of the tuple (the
-  "tag"). However, a different tag can be specified when defining a record:
+  By default, Elixir uses the record name as the first element of the tuple (the "tag").
+  However, a different tag can be specified when defining a record,
+  as in the following example we use `Customer` as the second argument of `defrecord/3`:
```

Comment:
Something is missing, I believe it would read more correctly as

> a different tag can be specified when defining a record, as in the following example, in which we use `Customer` as the second argument of `defrecord/3`